molgenis-semantic-mapper/src/main/java/org/molgenis/data/mapper/exception/MissingAttributeException.java

```diff
@@ -0,0 +1,35 @@
+package org.molgenis.data.mapper.exception;
+
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+
+// S2166 'Classes named like "Exception" should extend "Exception" or a subclass' often gives false positives at dev time
+@SuppressWarnings({ "squid:MaximumInheritanceDepth", "squid:S2166" })
+public class MissingAttributeException extends IncompatibleTargetException
+{
+	private static final String ERROR_CODE = "M04";
+	private final String attributeName;
+
+	public MissingAttributeException(String attributeName)
```

Comment:
Message says that it's missing in the target repository, perhaps update the class name to reflect that